ddl/ddl_api.go

```diff
@@ -533,8 +534,12 @@ func getDefaultValue(ctx sessionctx.Context, colName string, c *ast.ColumnOption
 			// its raw string content here.
 			return v.GetBinaryLiteral().ToString(), nil
 		}
-		// For other kind of fields (e.g. INT), we supply its integer value so that it acts as integers.
-		return v.GetBinaryLiteral().ToInt(ctx.GetSessionVars().StmtCtx)
+		// For other kind of fields (e.g. INT), we supply its integer as string value.
+		value, err := v.GetBinaryLiteral().ToInt(ctx.GetSessionVars().StmtCtx)
```

Comment:
It seems that the default value is `TypeBit`, why won't it go through line 528?

Reply:
The value type is `typeBit`, but the column type is `Int`. 
eg: `create table t (a int default b'1');`